/scripts/regression-net-connect.js:11
  throw error;
  ^

AggregateError [ECONNREFUSED]: 
    at internalConnectMultiple (node:net:r:c)
    at afterConnectMultiple (node:net:r:c)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:r:c)
    at TCPConnectWrap.emitInitNative (node:internal/async_hooks:r:c)
    at internalConnectMultiple (node:net:r:c)
    at TCP.emitInitNative (node:internal/async_hooks:r:c)
    at internalConnectMultiple (node:net:r:c)
    at afterConnectMultiple (node:net:r:c)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:r:c)
    at TCPConnectWrap.emitInitNative (node:internal/async_hooks:r:c)
    at internalConnectMultiple (node:net:r:c)
    at defaultTriggerAsyncIdScope (node:internal/async_hooks:r:c)
    at GetAddrInfoReqWrap.emitLookup [as callback] (node:net:r:c)
    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:r:c)
    at GetAddrInfoReqWrap.callbackTrampoline (node:internal/async_hooks:r:c)
    at TCP.emitInitNative (node:internal/async_hooks:r:c)
    at Socket.connect (node:net:r:c)
    at Object.connect (node:net:r:c)
    at Object.<anonymous> (/scripts/regression-net-connect.js:r:c)
    at Module._compile (node:internal/modules/cjs/loader:r:c)
    at Module._extensions..js (node:internal/modules/cjs/loader:r:c)
    at Module.load (node:internal/modules/cjs/loader:r:c)
    at Module._load (node:internal/modules/cjs/loader:r:c)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:r:c)
    at node:internal/main/run_main_module:r:c {
  code: 'ECONNREFUSED',
  [Symbol(originalCallSite)]: [ CallSite {}, CallSite {}, CallSite {} ],
  [Symbol(mutatedCallSite)]: [
    CallSite {}, CallSite {}, CallSite {},
    CallSite {}, CallSite {}, CallSite {},
    CallSite {}, CallSite {}, CallSite {},
    CallSite {}, CallSite {}, CallSite {},
    CallSite {}, CallSite {}, CallSite {},
    CallSite {}, CallSite {}, CallSite {},
    CallSite {}, CallSite {}, CallSite {},
    CallSite {}, CallSite {}, CallSite {},
    CallSite {}
  ],
  [errors]: [
    Error: connect ECONNREFUSED ::r:c
        at createConnectionError (node:net:r:c)
        at afterConnectMultiple (node:net:r:c)
        at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:r:c)
        at emitInitNative (node:internal/async_hooks:r:c)
        at emitInitScript (node:internal/async_hooks:r:c)
        at process.nextTick (node:internal/process/task_queues:r:c)
        at onDestroy (node:internal/streams/destroy:r:c)
        at Socket._destroy (node:net:r:c)
        at _destroy (node:internal/streams/destroy:r:c)
        at Socket.destroy (node:internal/streams/destroy:r:c)
        at Writable.destroy (node:internal/streams/writable:r:c)
        at internalConnectMultiple (node:net:r:c)
        at afterConnectMultiple (node:net:r:c)
        at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:r:c)
        at TCPConnectWrap.emitInitNative (node:internal/async_hooks:r:c)
        at internalConnectMultiple (node:net:r:c)
        at TCP.emitInitNative (node:internal/async_hooks:r:c)
        at internalConnectMultiple (node:net:r:c)
        at afterConnectMultiple (node:net:r:c)
        at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:r:c)
        at TCPConnectWrap.emitInitNative (node:internal/async_hooks:r:c)
        at internalConnectMultiple (node:net:r:c)
        at defaultTriggerAsyncIdScope (node:internal/async_hooks:r:c)
        at GetAddrInfoReqWrap.emitLookup [as callback] (node:net:r:c)
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:r:c)
        at GetAddrInfoReqWrap.callbackTrampoline (node:internal/async_hooks:r:c)
        at TCP.emitInitNative (node:internal/async_hooks:r:c)
        at Socket.connect (node:net:r:c)
        at Object.connect (node:net:r:c)
        at Object.<anonymous> (/scripts/regression-net-connect.js:r:c)
        at Module._compile (node:internal/modules/cjs/loader:r:c)
        at Module._extensions..js (node:internal/modules/cjs/loader:r:c)
        at Module.load (node:internal/modules/cjs/loader:r:c)
        at Module._load (node:internal/modules/cjs/loader:r:c)
        at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:r:c)
        at node:internal/main/run_main_module:r:c {
      errno: -61,
      code: 'ECONNREFUSED',
      syscall: 'connect',
      address: '::1',
      port: 24075
    },
    Error: connect ECONNREFUSED 127.0.0.1:24075
        at createConnectionError (node:net:r:c)
        at afterConnectMultiple (node:net:r:c)
        at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:r:c)
        at emitInitNative (node:internal/async_hooks:r:c)
        at emitInitScript (node:internal/async_hooks:r:c)
        at process.nextTick (node:internal/process/task_queues:r:c)
        at onDestroy (node:internal/streams/destroy:r:c)
        at Socket._destroy (node:net:r:c)
        at _destroy (node:internal/streams/destroy:r:c)
        at Socket.destroy (node:internal/streams/destroy:r:c)
        at Writable.destroy (node:internal/streams/writable:r:c)
        at internalConnectMultiple (node:net:r:c)
        at afterConnectMultiple (node:net:r:c)
        at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:r:c)
        at TCPConnectWrap.emitInitNative (node:internal/async_hooks:r:c)
        at internalConnectMultiple (node:net:r:c)
        at TCP.emitInitNative (node:internal/async_hooks:r:c)
        at internalConnectMultiple (node:net:r:c)
        at afterConnectMultiple (node:net:r:c)
        at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:r:c)
        at TCPConnectWrap.emitInitNative (node:internal/async_hooks:r:c)
        at internalConnectMultiple (node:net:r:c)
        at defaultTriggerAsyncIdScope (node:internal/async_hooks:r:c)
        at GetAddrInfoReqWrap.emitLookup [as callback] (node:net:r:c)
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:r:c)
        at GetAddrInfoReqWrap.callbackTrampoline (node:internal/async_hooks:r:c)
        at TCP.emitInitNative (node:internal/async_hooks:r:c)
        at Socket.connect (node:net:r:c)
        at Object.connect (node:net:r:c)
        at Object.<anonymous> (/scripts/regression-net-connect.js:r:c)
        at Module._compile (node:internal/modules/cjs/loader:r:c)
        at Module._extensions..js (node:internal/modules/cjs/loader:r:c)
        at Module.load (node:internal/modules/cjs/loader:r:c)
        at Module._load (node:internal/modules/cjs/loader:r:c)
        at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:r:c)
        at node:internal/main/run_main_module:r:c {
      errno: -61,
      code: 'ECONNREFUSED',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 24075
    }
  ]
}

Node.js v20.15.1
